lib/datadog/appsec/contrib/rack/ext.rb

```diff
@@ -1,13 +1,26 @@
 # typed: ignore
+# frozen_string_literal: true
 
 module Datadog
   module AppSec
     module Contrib
       module Rack
         # Rack integration constants
         module Ext
-          APP = 'rack'.freeze
-          ENV_ENABLED = 'DD_TRACE_RACK_ENABLED'.freeze # TODO: DD_APPSEC?
+          APP = 'rack'
+          ENV_ENABLED = 'DD_TRACE_RACK_ENABLED' # TODO: DD_APPSEC?
```

Comment:
Is this actually used? If not, unused constants should be removed.

Reply:
They are use in `lib/datadog/appsec/contrib/rack/configuration/settings.rb`